orienteer-core/src/main/java/org/orienteer/core/component/property/OLocalizationEditPanel.java

```diff
@@ -27,7 +27,6 @@ protected Component getKeyEditComponent(ListItem<EmbeddedMapEditPanel.Pair<V>> i
         Select2Choice<String> select2 = new Select2Choice<String>("key", new PropertyModel<String>(item.getModel(), "key"), LanguagesChoiceProvider.INSTANCE);
         select2.getSettings().setCloseOnSelect(true).setTheme(OClassMetaPanel.BOOTSTRAP_SELECT2_THEME);
         select2.add(new AttributeModifier("style", "width: 100%"));
-        select2.setConvertEmptyInputStringToNull(false);
```

Comment:
Why this is needed? From my perspective "" should be treated as null, so this line should remain.

Reply:
This is need, because `wicket-stuff` removed this methods in `8.3.0`